play_sudoku.py

```diff
@@ -68,7 +68,7 @@ def inside_board(x: int, y: int) -> bool:
 
 def inside_accept(x: int, y: int) -> bool:
     """
-    Check whether the coordinates are inside the accept button.
+    Check whether the coordinates are inside the reject button.
```

Comment:
This docstring belongs to `inside_accept`. I think you wanted to change the docstring of `inside_reject` on line 82.